extensions/wikia/AdEngine/js/slot/service/srcProvider.js

```diff
@@ -35,7 +45,7 @@ define('ext.wikia.adEngine.slot.service.srcProvider',  [
 		if (adContext.get('opts.premiumOnly') && !adContext.get('opts.isAdTestWiki')) {
 			originalSrc = 'premium';
 		}
-		if (isRecoverableByPF(extra) || isRecoverableByIL()) {
+		if (isRecoveryAllowedByBab(extra) && (isRecoverableByPF(extra) || isRecoverableByIL())) {
```

Comment:
I believe you don't need to add additional check here. `instartLogic.isBlocking()` won't return false when it's not enabled. The same should be valid for PF.